src/components/accordion/accordion.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+    <div class="ui-accordion" role="heading" :class="rootClasses">
+        <button type="button"
+                class="u-reset ui-accordion--head"
+                v-if="!!$scopedSlots.head"
+                :aria-expanded="state.isOpen ? 'true' : 'false'"
+                @click.prevent="onRequestChange(!state.isOpen)">
+            <slot name="head" :is-open="state.isOpen"/>
+        </button>
+        <div class="ui-accordion--body"
+                role="presentation"
+                ref="body"
+                @transitionend="onTransitionEnd">
+            <slot name="body" :is-open="state.isOpen" :is-visible="isVisible"/>
+        </div>
+    </div>
+</template>
+
+<script>
+    import forceReflow from 'bianco.force-reflow';
```

Comment:
Maybe we should add bianco as a whole. Else we need to add every single module in the webpack config.

Reply:
I have update the code using only one bianco dependency